anatomy/tracks/concept-exercises.md

```diff
@@ -255,7 +267,9 @@ This file contains meta information on the exercise:
   - Including reviewers if their reviews are meaningful/actionable/actioned.
 - `forked_from`: Which exercise(s) it was forked from (required if the exercise is forked)
 - `files`: The files used in this exercises (keys for `solution`, `test`, and `exemplar`, which point to the locations of the files specified in the "Stub implementation", "Tests", and "Examplar Implementation" sections listed below)
-- `language_versions` Language version requirements (optional)
+- `language_versions`: Language version requirements (optional)
+- `source`: The source this exercise is based on (optional)
+- `source_url`: The URL of the source this exercise is based on (optional)
```

Comment:
These are the new fields to describe the source of an exercise. They match the names used in prob-specs metadata: https://github.com/exercism/problem-specifications/blob/main/exercises/zebra-puzzle/metadata.yml